Tests/xcprojTests/PBXProjWriterSpec.swift

```diff
@@ -36,10 +36,12 @@ class PBXProjEncoderSpec: XCTestCase {
         let quote = "\""
         let escapedNewline = "\\n"
         let escapedQuote = "\\\""
+        let escapedEscape = "\\\\"
+        let escapedTab = "\\t"
 
         let values: [String: String] = [
             "a": "a",
-            "a".quoted: "a".quoted,
+            "a".quoted: "\(escapedQuote)a\(escapedQuote)".quoted,
```

Comment:
Like the below cases this is escaping the surrounding quotes when it doesn’t need to. This would cause even more diffs when a project with any quoted values is opened and then saved, as this would happen every time

Reply:
We need to escape quotes that have been inserted be user. This can bee seen in https://github.com/xcodeswift/xcproj/issues/71. It wont add additional quotes because the quotes that are meant for encoding are removed during decoding.